tests/test_gluon_block.py

```diff
@@ -50,6 +51,7 @@ def forward(self, x):
     out.wait_to_read()
 
 
+@pytest.mark.skip('Skip due to MXNet bug https://github.com/apache/incubator-mxnet/issues/19659')
```

Comment:
is it still worth running? we can also mark it as xfail.

Reply:
It's worth running and may appear if we try to hybridize network with boolean mask.

Reply:
In that case I would recommend changing it to xfail mark, so that the test would be run (and expected to fail for now)